### PR TITLE
Migrate script added in PR #212 to use auto lines

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

File: tests/stub/routing/scripts/v3_no_routing/optional_hello.script [hunk 1]
```diff
@@ -1,6 +1,7 @@
 !: BOLT #VERSION#
-!: AUTO RESET
-!: AUTO GOODBYE
 
+?: GOODBYE
```
Comment (injectives) [inline tests/stub/routing/scripts/v3_no_routing/optional_hello.script:3]: Is this expected here?

File: tests/stub/routing/scripts/v4x1_no_routing/optional_hello.script [hunk 2]
```diff
@@ -1,6 +1,7 @@
 !: BOLT #VERSION#
-!: AUTO RESET
-!: AUTO GOODBYE
 
+?: GOODBYE
```
Comment (injectives) [inline tests/stub/routing/scripts/v4x1_no_routing/optional_hello.script:3]: Is this expected here? (same as the other comment)

File: tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script [hunk 3]
```diff
@@ -1,13 +1,8 @@
...
-!: AUTO GOODBYE
 
 C: HELLO {"scheme": "basic", "credentials": "c", "principal": "p", "user_agent": "#USER_AGENT#" #ROUTING# #EXTRA_HELLO_PROPS#}
 S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
-# Added this block for GO
-{?
-    C: RESET
-    S: SUCCESS {}
-?}
+*: RESET
```
Comment (injectives) [inline tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script:5]: Wouldn't `?: RESET` be sufficient?

File: tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script [hunk 4]
```diff
@@ -29,7 +24,5 @@ S: SUCCESS {"server": "#SERVER_AGENT#", "connection_id": "bolt-123456789"}
        RECORD [7]
        SUCCESS {}
 }}
-{?
-    C: RESET
-    S: SUCCESS {}
-?}
+*: RESET
```
Comment (injectives) [inline tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script:27]: Wouldn't ?: RESET be sufficient? (duplicate question)

File: tests/stub/routing/scripts/v3_no_routing/optional_hello.script (hunk 1 above)
Comment (robsdedude) [reply tests/stub/routing/scripts/v3_no_routing/optional_hello.script:3]: JS seems a bit rude ;) and doesn't even say `HELLO` to figure out if the server has multi db support. Arguable it should because unauthenticated connections will be terminated after a while on a real server, if I'm not mistaken. @bigmontz thoughts on this?

File: tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script (hunk 3 above)
Comment (robsdedude) [reply tests/stub/routing/scripts/v3_no_routing/writer_yielding_multiple_records.script:5]: It would probably suffice for all official drivers. Who knows about community drivers (also those that might start using TestKit in the future). Also, we have separate tests in the optimization folder that assert that a driver only sends the bare minimum number of resets. In ever other test, I suggest, we shouldn't make assertions on how often a driver wants to reset as long as the resets happen at an appropriate point in the script.

File: tests/stub/routing/scripts/v3_no_routing/optional_hello.script (hunk 1 above)
Comment (bigmontz) [reply tests/stub/routing/scripts/v3_no_routing/optional_hello.script:3]: Yes, JS Driver does these checks in `No time, brother` mode. It creates the connection, handshakes, checks the version and closes.

**Review by injectives — APPROVED** [review]
LGTM
